src/app/Main/Body/QueryPage/QueryPageContent/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -27,11 +29,26 @@ export function QueryPageParameterContentComponent(props: {
         const elements: JSX.Element[] = [];
         Object.keys(selectedFilters).forEach((filterElement) => {
             if (selectedFilters[filterElement].length !== 0) {
+                let isSubFilter = false;
```

Comment:
As stated in another comment: Messing with the sub filters is clearly a part of a container. This component is just for displaying parameters and not for putting the data together it should display.

Reply:
This is part of the container refactoring ticket.